src/Snippet/Target.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Snippet;
+
+use ReflectionClass;
+
+/**
+ * Bolt Snippet target location.
+ *
+ * This class categorizes all possible snippet locations in constants.
+ */
+class Target
+{
+    // unpredictable
```

Comment:
why we have unpredictable targets?

Reply:
Because the HTML structure is unknown.. If the implementor put his `<script>` files at the bottom of `<body>`, that's where `AFTER_JS` will go. However if the implementor put his `<script>` files in the `<head>` somewhere, that's where it will be put.